hack/update-import-aliases.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${SCRIPT_ROOT}"
+
+ROOT_PATH=$(pwd)
+GOPATH=$(go env GOPATH | awk -F ':' '{print $1}')
+
+temp_path=$(mktemp -d)
+pushd "${temp_path}" >/dev/null
+cp "${ROOT_PATH}"/go.mod .
+GO111MODULE=on go get "k8s.io/kubernetes@v1.21.3"
+popd >/dev/null
+
+IMPORT_ALIASES_PATH="${ROOT_PATH}/hack/.import-aliases"
+INCLUDE_PATH="(cmd|pkg|test/e2e|test/helper)"
+
+go run "$GOPATH"/pkg/mod/k8s.io/kubernetes@v1.21.3/cmd/preferredimports/preferredimports.go -confirm -import-aliases "${IMPORT_ALIASES_PATH}" -include-path "${INCLUDE_PATH}" "${ROOT_PATH}"
```

Comment:
We can run `preferredimports` directly now if we have installed the command. like 
```
preferredimports xxx
```

Reply:
OK.